py3status/modules/exchange_rate.py

```diff
@@ -46,14 +46,13 @@ def post_config_hook(self):
 
     def rates(self):
         try:
-            result = self.py3.request(
+            data = self.py3.request(
                 URL, timeout=self.request_timeout
-            )
+            ).json()
         except self.py3.RequestException:
-            result = None
-        rates = {}
-        if result:
-            data = result.json()
+            data = None
+        if data:
+            rates = {}
```

Comment:
`rates` is already defined as the name of the function, please rename it since we're doing QA work here

Reply:
Renamed the func. :+1: